micro-ui/web/micro-ui-internals/packages/modules/pgr/src/pages/employee/CreateComplaint/index.js

```
@@ -178,7 +163,7 @@ export const CreateComplaint = ({ parentUrl }) => {
   //On SUbmit
   const onSubmit = async (data) => {
     if (!canSubmit) return;
-    const cityCode = selectedCity.code;
+    const cityCode= Digit.Utils.getMultiRootTenant() ? Digit.ULBService.getStateId() : cityCode;
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)